include/libtorrent/load_torrent.hpp

```diff
@@ -28,14 +28,20 @@ namespace libtorrent {
 	//   * added to a session via add_torrent() or async_add_torrent()
 	//   * saved as a .torrent_file via write_torrent_file()
 	//   * turned into a magnet link via make_magnet_uri()
+	TORRENT_EXPORT add_torrent_params load_torrent_file(
+		std::string const& filename, error_code& ec, load_torrent_limits const& cfg);
 	TORRENT_EXPORT add_torrent_params load_torrent_file(
 		std::string const& filename, load_torrent_limits const& cfg);
 	TORRENT_EXPORT add_torrent_params load_torrent_file(
 		std::string const& filename);
+	TORRENT_EXPORT add_torrent_params load_torrent_buffer(
+		span<char const> buffer, error_code& ec, load_torrent_limits const& cfg);
 	TORRENT_EXPORT add_torrent_params load_torrent_buffer(
 		span<char const> buffer, load_torrent_limits const& cfg);
 	TORRENT_EXPORT add_torrent_params load_torrent_buffer(
 		span<char const> buffer);
+	TORRENT_EXPORT add_torrent_params load_torrent_parsed(
+		bdecode_node const& torrent_file, error_code& ec, load_torrent_limits const& cfg);
```

Comment:
these new overloads should be covered by `test/test_torrent_info.cpp` as well.

Reply:
added only for `load_torrent_file` (similar to the existing), since it use the others.